external/external.sh

```diff
@@ -180,11 +180,19 @@ if [ $command_type == "run" ]; then
 	fi
 	if [ $reportsrc != "false" ]; then
 		echo "docker run $mountV --name $test-test adoptopenjdk-$test-test:${JDK_VERSION}-$package-$docker_os-${JDK_IMPL}-$build_type $testtarget"
-		docker run $mountV --name $test-test adoptopenjdk-$test-test:${JDK_VERSION}-$package-$docker_os-${JDK_IMPL}-$build_type $testtarget;
+		if [ -n "$testtarget" ]; then
+			docker run $mountV --name $test-test adoptopenjdk-$test-test:${JDK_VERSION}-$package-$docker_os-${JDK_IMPL}-$build_type "$testtarget";
+		else
+			docker run $mountV --name $test-test adoptopenjdk-$test-test:${JDK_VERSION}-$package-$docker_os-${JDK_IMPL}-$build_type;
+		fi
```

Comment:
I am not very clear why we need if-else here

Reply:
As $testtarget may be the string include spaces we have to use "$testtarget" as parameter, which also means it could be an empty string. In that case the empty string may override the default setting, which is not what we expected. We could also try to test if $testtarget is null instead of an empty string, but I don't think that helps a lot.